src/borrow_tracker/tree_borrows/wildcard.rs

```diff
@@ -406,9 +403,13 @@ impl Tree {
         let protected = protected_tags.contains_key(&tag);
         // TODO: Only initialize neccessary ranges.
```

Comment:
Is this comment worth keeping? Doesn't seem super useful to me. Better comment on what is happening.